### ISSUE TITLE
Loader Duplication with fortune

### ISSUE THREAD

**Issue by CrashWho** [BG-Software-LLC/WildLoaders#105]
### Minecraft's Version

This server is running Paper version 1.21-122-master@58c7ea3 (2024-07-30T19:05:08Z) (Implementing API version 1.21-R0.1-SNAPSHOT)

### Plugin's Version

2024.1-b30

### Describe the bug

When i try to break chunk loader block with a pickaxe with fortune, it still duplicates it

### To Reproduce

you have to break chunk loader block with fortune

**Comment by OmerBenGera** [comment]
I cannot seem to replicate the issue

**Comment by CrashWho** [comment]
I'm using a custom enchantment plugin and a enchantment called "telepathy" which should give me the item in the inventory but it duplicates the chunk loader

**Comment by CrashWho** [comment]
I just turned off the enchant and everything works fine now

**Comment by OmerBenGera** [comment]
> I just turned off the enchant and everything works fine now, but the problem was only with the chunk loader block.

Yeah sounds right, probably because WildLoaders handles the breaking of the loader block by itself.
Can you contact the developer of the enchants plugin and send him a link to this issue? I am sure him and I can somehow fix this conflict together.

**Comment by CrashWho** [comment]
yeah sure ill conctact him

**Comment by CrashWho** [comment]
anyway i just created an addon with your api to make the chunk loader instantly destroyed when left clicked so i have no problem with custom enchants

**Comment by OmerBenGera** [comment]
Great.
However, thats not a good solution for the long run, therefore I'll keep this opened for now

**Comment by lDoomerl** [comment]
I have the same issue, usual Luck enchantment can dupe loaders

**Comment by OmerBenGera** [comment]
> I have the same issue, usual Fortune enchantment can dupe loaders

Probably another plugin causing this, as the plugin overrides the vanilla mechanism of breaking these blocks

**Comment by OmerBenGera** [comment]
Any news regarding this?

**Comment by OmerBenGera** [comment]
I am closing this due to lack of response
If the issue still occurs, let me know

**Comment by iCodeYT** [comment]
This still happens

**Comment by OmerBenGera** [comment]
> This still happens, doesn't seem to be happening with any other plugins, just wild loaders, and same, It happens with the custom enchant called "telepathy" by AdvancedEnchantments.

Open a new bug report please
